packages/build/src/download-center.ts

```diff
@@ -72,6 +72,12 @@ const CONFIG = `
           "name": "Debian 64-bit",
           "download_link": "https://downloads.mongodb.com/compass/mongosh_{{version}}_amd64.deb"
         }
+        {
+          "arch": "x64",
+          "os": "rhel",
+          "name": "Redhat 64-bit",
+          "download_link": "https://downloads.mongodb.com/compass/mongosh-{{version}}-x86_64.rpm"
```

Comment:
I want to double check that we are ok building this for this `arch`, and this naming for the downloads centre package is ok. If anyone has thoughts, please let me know.